index.js

```diff
@@ -31,6 +31,25 @@ $(document).ready(function() {
     });
 });
 
+// opens modal when img is clicked
```

Comment:
Various style things (I know, I'm picky haha)
- Always use single quotes for strings
    - (Yes, I know the FAQ code doesn't use them; we'll refactor that later)
- Single space after `//` and first word of comment capitalized
- Space after func params, and before each brace: i.e., `'click', 'img', function() {`
- Remember semicolons! (we might take these all out later since iirc parcel includes ES6 support which frees us from needing semicolons)

Also thank you for doing a full if statement! Programmers these days are so eager to remove the curly brace for a single `if`